praetorian_cli/ui/hunt/app.py

```
@@ -0,0 +1,551 @@
+            hours = _parse_duration_int(self._start_duration)
+            expires_at = (datetime.now(timezone.utc) + timedelta(hours=hours)).strftime('%Y-%m-%dT%H:%M:%SZ')
+            body = {'prompt': self._start_prompt, 'expiresAt': expires_at}
+            if self._start_scope:
+                body['scope'] = self._start_scope
+
+            result = self.sdk.post('hunt', body)
+            self.hunt_uuid = result.get('uuid', '')
+            self.hunt_status = result.get('status', 'active')
+            self._update_context_bar()
+
+            log.write(f"[bold green]Hunt launched![/bold green]")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)